source/_lovelace/picture.markdown

```diff
@@ -18,7 +18,7 @@ type:
   type: string
 image:
   required: true
-  description: The URL of an image.
+  description: The URL of an image. When using files on your filesystem use the /local path. This points to your the www subfolder of your config folder. By default the www folder does not exist and you have to create this yourself and restart Home Assistant. Example path /local/filename.jpg
```

Comment:
Should we instead link to the part of the documentation that explains this?

<https://www.home-assistant.io/integrations/http/#hosting-files>

Reply:
Good feedback, I did not know that page yet (used the forums). I've change the text to include this link. And will re-commit that